src/swarm/neo/client/mixins/BatchRequestCore.d

```diff
@@ -30,6 +30,83 @@ import swarm.neo.client.mixins.AllNodesRequestCore;
 
 /*******************************************************************************
 
+    Helper function providing the standard logic for connecting a request-on-
+    conn of a suspendable all-nodes batch request. To be called from the
+    Connector policy instance of a BatchRequest.
```

Comment:
I _think_ this should say "To be called from the Connector policy instance of an AllNodesRequest."